README.md

```diff
@@ -20,13 +20,49 @@ module.exports = {
       options: {
         accessToken: 'YOUR_TOKEN',
         version: 'draft',
+        localAssets: true, // Optional parameter to download the images to use with Gatsby Image Plugin
         languages: ['de', 'at'] // Optional parameter. Omission will retrieve all languages by default.
       }
     }
   ]
 }
 ```
 
+### With Gatsby's image
+
+You need to set the `localAssets` option to `true`. Here is an example of the usage:
+
+```
+import { graphql } from "gatsby"
+import { GatsbyImage, getImage } from "gatsby-plugin-image"
+
+function BlogPost({ data }) {
+  const image = getImage(data.file)
+  return (
+    <section>
+      <h2>{data.blogPost.title}</h2>
+      <GatsbyImage image={image} />
+    </section>
+  )
+}
+
+export const pageQuery = graphql`
+  query {
+    file(name: {eq: "demo"}) {
```

Comment:
Can we have an example where we query all pages like this:
```
            pages: allStoryblokEntry(
              filter: { field_component: { eq: "Page" } }
            ) {
              edges {
                node {
                  id
                  name
                  slug
                  field_component
                  full_slug
                  content
                }
              }
            }
```
and the content contains some images (the asset field with the image). How can we convert this image asset to the GatsbyImage? 

Inspired by this plugin and discussion: https://github.com/bejamas/gatsby-storyblok-image/issues/24

Reply:
The problem with this is that the content coming from Storyblok is very dynamic, which is hard to type for the static Gatsby GraphQL layer.  With this PR we aim to first support the gatsby images without any breaking changes for current users and then maybe in the future we will rewrite the plugin to allow dynamic support of the content